res/database/Default/user/kasi.xml

```diff
@@ -1243,7 +1243,7 @@
 			<studio_size min="1" max="2" slope="60" />
 		</scripttemplate>
 
-		<!--Episoden gekürzt, kein optionaler Kult-->
+		<!--Episoden gekürzt (TODO: Klären Warum?), kein optionaler Kult-->
 		<scripttemplate licence_type="3" product="2" guid="Kasi-script-Metalsucher">
```

Comment:
On the look for "Metal" (music) or "Metall" (the resource).

Think it got cut down to avoid a 20 episodes series.

Reply:
Only 3 episodes were left out. Feel free to add them. A general review for episodes would be nice at some point (more variety in titles and descriptions).

Reply:
> Feel free to add them

ok, but at a later date. Requires a closer look. The episode content seems to overlap.